arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1967,12 +2180,32 @@
       },
       "description": "Message Count Details."
     },
+    "TopicCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
```

Comment:
Not sure if we should be able to update the resource properties like name or location, will need to verify